kyuubi-common/src/test/scala/org/apache/kyuubi/KyuubiFunSuite.scala

```diff
@@ -35,9 +35,9 @@ trait KyuubiFunSuite extends FunSuite
   with ThreadAudit
   with Logging {
   // scalastyle:on
```

Comment:
not relate about this pr, why here forbid scalastyle check ？

Reply:
Let me try to remove it.

Reply:
The forbid is necessary because of the rule:
```
  <check customId="funsuite" level="error" class="org.scalastyle.scalariform.TokenChecker" enabled="true">
    <parameters><parameter name="regex">^FunSuite[A-Za-z]*$</parameter></parameters>
    <customMessage>Tests must extend org.apache.kyuubi.KyuubiFunSuite instead.</customMessage>
  </check>
```